source/common/upstream/health_checker_impl.h

```diff
@@ -18,9 +18,10 @@
 
 #include "common/common/logger.h"
 #include "common/http/codec_client.h"
-#include "common/json/json_loader.h"
```

Comment:
I've fixed a few more in the outlier detection and upstream impl, please let me know if you see any other places I missed.